### PR TITLE
Update syncplay to 1.6.8 and fixed offenses with brew style --fix

### PR DESCRIPTION
**Pull request by albertoxamin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

**Comment by albertoxamin** [comment]
Why is the workflow trying to get the beta isn't it supposed to be only for stable versions?

**Comment by SMillerDev** [comment]
The livecheck is pretty dumb by default and only checks for the highest numbered version. Maybe @cho-m or @nandahkrishna can help refine it here.

**Review by nandahkrishna — COMMENTED** [review]
You'll need to add a `livecheck` block as this is picking up Git tags that are prereleases:
```
livecheck do
  url :stable
  regex(/^v?(\d+(?:\.\d+)+)$/)
end
```

File: Casks/syncplay.rb [hunk 1]
```diff
@@ -13,7 +13,10 @@
   depends_on macos: ">= :sierra"
 
   app "Syncplay.app"
-
+  livecheck do
```
[the remark this answers is not in the document]
Comment (nandahkrishna) [reply Casks/syncplay.rb:16]: Yes, you'll have to fix the style issues though. `brew style --fix syncplay`.

[the remark this answers is not in the document]
Comment (albertoxamin) [reply Casks/syncplay.rb:13]: It is done

File: Casks/syncplay.rb [hunk 2]
```diff
@@ -1,12 +1,20 @@
+# typed: false
+# frozen_string_literal: true
+
```
Comment (SMillerDev) [inline Casks/syncplay.rb:3]:
```suggestion
```
 these aren't actually needed.

**Review by SMillerDev — APPROVED** [review]
Thanks @albertoxamin  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request merged**